spec/models/target_spec.rb

```diff
@@ -0,0 +1,36 @@
+# == Schema Information
+#
+# Table name: targets
+#
+#  id         :bigint           not null, primary key
+#  latitude   :float            not null
+#  longitude  :float            not null
+#  radius     :float            not null
+#  title      :string           not null
+#  created_at :datetime         not null
+#  updated_at :datetime         not null
+#  topic_id   :bigint           not null
+#  user_id    :bigint           not null
+#
+# Indexes
+#
+#  index_targets_on_topic_id  (topic_id)
+#  index_targets_on_user_id   (user_id)
+#
+require 'rails_helper'
+
+RSpec.describe Target, type: :model do
+  describe 'Validations' do
```

Comment:
```suggestion
  describe 'validations' do
```

Reply:
I prefer using camel case only for classes, @mrodriguez90 wdyt about this?

Reply:
I agree, in rails you are going to use camel case only if the syntax asks you for it.
This is the way, mando!